gandalf.js

```diff
@@ -2,16 +2,17 @@
  * Extras Plugin
```

Comment:
SCRIPT-Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.